### ISSUE TITLE
MuPDF crashes on *invalid* files (DjVu, PDF)

### ISSUE THREAD

**Issue by baskerville** [koreader/koreader#2335]
Trying to open an *invalid* DJVU file will result in a crash/reboot:
```
error: unknown image file format
*** glibc detected *** ./luajit: double free or corruption (out): 0x2be5cdf8 ***
Aborted
```

**Comment by houqp** [comment]
Looks like an upstream bug. Can you provide the djvu file for test?

**Comment by baskerville** [comment]
You can create an invalid DJVU file with `touch abc.djvu`.

**Comment by robert00s** [comment]
Currently, the problem no longer exist. 
When we try to open invalid (touch invalid.djvu or touch invalid.pdf) pdf and djvu files we get the following message:
![zaznaczenie_039](https://user-images.githubusercontent.com/22982594/34322518-4ec3873e-e829-11e7-8dbd-919c314c5930.png)

**Comment by Frenzie** [comment]
Alright, let's say fixed in #2956 in lieu of a better testcase (since empty file is a very specific subset of invalid file) and open a new issue or reopen should anything specific come up.
